src/ApiCheck.Task/ApiCheckTask.cs

```diff
@@ -117,66 +62,15 @@ protected override bool ValidateParameters()
         }
 
         /// <inheritdoc />
-        protected override string GenerateFullPathToTool()
-        {
-            // ToolExe equals ToolName by default. Assume (as base class does) any user override is in the same directory.
-            if (Framework.StartsWith("net4", StringComparison.OrdinalIgnoreCase))
-            {
-                var taskAssemblyFolder = Path.GetDirectoryName(GetType().GetTypeInfo().Assembly.Location);
-                return Path.GetFullPath(Path.Combine(taskAssemblyFolder, "..", "net46", ToolExe));
-            }
-
-            // If muxer does not find dotnet, fall back to system PATH and hope for the best.
-            return DotNetMuxer.MuxerPath ?? ToolExe;
-        }
-
-        /// <inheritdoc />
         protected override string GenerateCommandLineCommands()
         {
-            var arguments = string.Empty;
-            if (!Framework.StartsWith("net4", StringComparison.OrdinalIgnoreCase))
-            {
-                var taskAssemblyFolder = Path.GetDirectoryName(GetType().GetTypeInfo().Assembly.Location);
-                var toolPath = Path.Combine(taskAssemblyFolder, "..", "netcoreapp2.0", ApiCheckToolName + ".dll");
-                arguments = $@"""{Path.GetFullPath(toolPath)}"" ";
-            }
-
-            arguments += "compare";
-            if (ExcludePublicInternalTypes)
-            {
-                arguments += " --exclude-public-internal";
-            }
-
-            arguments += $@" --assembly ""{AssemblyPath}"" --framework {Framework}";
-            arguments += $@" --project ""{ProjetAssetsPath}"" --api-listing ""{ApiListingPath}""";
+            var arguments = base.GenerateCommandLineCommands();
```

Comment:
Yes, will need a `GenerateCommandLineCommands()` override in `ApiCheckGenerateTask`.